### PR TITLE
Add cmd line option to switch server config

### PR DESCRIPTION
**Pull request by dcolligan**

### CODE REVIEW

**Comment by Naburimannu** [comment]
+1

**Comment by hershman** [comment]
+1

File: ga4gh/cli.py [hunk 1]
```diff
@@ -58,6 +61,7 @@ def server_main():
     if "variantSetClass" not in args:
         parser.print_help()
     else:
+        frontend.configure(args)
```
Comment (kerrydc) [inline ga4gh/cli.py]: This only needs the config arg, not args.

File: ga4gh/frontend.py [hunk 2]
```diff
@@ -13,11 +13,14 @@
 import flask.ext.cors as cors
 
 app = api.FlaskAPI(__name__)
-app.config.from_object('ga4gh.server.config:DefaultConfig')
-if os.environ.get('GA4GH_CONFIGURATION') is not None:
-    app.config.from_envvar('GA4GH_CONFIGURATION')
 
-cors.CORS(app, allow_headers='Content-Type')
+
+def configure(args):
```
Comment (kerrydc) [inline ga4gh/frontend.py]: ditto here.

**Comment by jeromekelleher** [comment]
+1, with @kerrydc's requests.

File: ga4gh/frontend.py (hunk 2 above)
Comment (dcolligan) [inline ga4gh/frontend.py]: If a config can not be found the app.config.from_object(configStr) will fail with "ImportError: No module named DummyConfig," so implicitly that's already taken care of.

**Comment by dcolligan** [comment]
Pushed again with @kerrydc's suggestions incorporated

**Comment by jeromekelleher** [comment]
Looks good to me. Let's wait to see what @kerrydc says before we merge.

**Comment by pashields** [comment]
+1

**Pull request merged**